core/src/templates/components/card/card.twig

```diff
@@ -153,13 +164,16 @@
             </a>
           {%- endif -%}
         </h2>
+        
       {% endif -%}
     {% endblock -%}
 
-    {%- block block_card_body %}
-      {# An open field for body content #}
-      {{ card_body }}
-    {% endblock -%}
+    {# An open field for body content #}
+    {% if card_body is not empty %}
```

Comment:
If statement is not needed here as card_body won't render anything if it is empty anyhow.